README.md

```diff
@@ -200,7 +200,7 @@ at present for doing this.
 
 ### Stamping
 
-The first option is to use `stamp = True`.
+The first option is to use stamping.
```

Comment:
can you update these docs to indicate explicitly how the new inferred stamping works